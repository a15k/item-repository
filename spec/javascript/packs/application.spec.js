import renderer from 'react-test-renderer';
import Application from 'packs/application';

jest.mock('models/user', () => ({
  isMember: true,
  username: 'bobby tables',
}));

describe(Application, () => {

  it('renders and matches snapshot', () => {
    const app = renderer.create(<Application />);
    expect(app.toJSON()).toMatchSnapshot();
    app.unmount();
  });

});
