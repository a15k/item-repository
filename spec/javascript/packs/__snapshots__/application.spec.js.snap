// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A15KApplication renders and matches snapshot 1`] = `
.c0 {
  margin-bottom: 1rem;
}

.c1 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c2 {
  margin-left: 1rem;
}

<div
  className="a15k-root"
>
  <nav
    className="navbar navbar-dark bg-dark c0"
  >
    <div
      className="navbar-brand c1"
    >
      Assessment Network
    </div>
    <div
      className="c2 dropdown"
      onKeyDown={[Function]}
    >
      <button
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle btn btn-secondary"
        onClick={[Function]}
        type="button"
      >
        Actions
      </button>
      <div
        aria-hidden={true}
        className="dropdown-menu dropdown-menu-right"
        role="menu"
        tabIndex="-1"
      >
        <a
          aria-current={null}
          className="dropdown-item"
          href="/search"
          onClick={[Function]}
          style={undefined}
        >
          Search
        </a>
        <a
          aria-current={null}
          className="dropdown-item"
          href="/formats"
          onClick={[Function]}
          style={undefined}
        >
          Formats
        </a>
      </div>
    </div>
    <div
      className="c2 dropdown"
      onKeyDown={[Function]}
    >
      <button
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle btn btn-secondary"
        onClick={[Function]}
        type="button"
      >
        bobby tables
      </button>
      <div
        aria-hidden={true}
        className="dropdown-menu dropdown-menu-right"
        role="menu"
        tabIndex="-1"
      >
        <button
          className="dropdown-item"
          href={undefined}
          onClick={[Function]}
          tabIndex="0"
          type="button"
        >
          Profile
        </button>
        <button
          className="dropdown-item"
          onClick={[Function]}
          tabIndex="0"
          type="button"
        >
          Logout
        </button>
      </div>
    </div>
  </nav>
  <div
    className="container"
  >
    <h3>
      Not Found
    </h3>
  </div>
  <form
    action="/accounts/logout"
    id="logout-form"
    method="post"
  >
    <input
      name="authenticity_token"
      type="hidden"
      value={undefined}
    />
    <input
      name="_method"
      type="hidden"
      value="delete"
    />
  </form>
</div>
`;
